### ISSUE TITLE
Cover replacing designtime services like pluralization

### ISSUE THREAD

**Issue by AndriySvyryd** [aspnet/EntityFramework.Docs#415]

**Comment by bricelam** [comment]
```C#
class MyDesignTimeServices : IDesignTimeServices
{
    public void ConfigureDesignTimeServices(IServiceCollection services)
        => services.AddSingleton<IPluralizer, MyPluralizer>();
}
```

**Comment by divega** [comment]
I added something to https://docs.microsoft.com/en-us/ef/core/what-is-new/index#pluralization-hook-for-dbcontext-scaffolding. Not a replacement for having a proper section on this.

**Comment by divega** [comment]
Fixed in 28c941ca7ffe06194904a326f5fda0e6776171aa.
